src/js/components/help/downloads.js

```diff
@@ -0,0 +1,324 @@
+
+const cookies = new Cookies();
+
+const osMap = {
+  MacOs: 'darwin',
+  Unix: 'linux',
+  Linux: 'linux'
+};
+
+const defaultArchitectures = ['armhf', 'arm64', 'amd64'];
+const defaultOSVersions = ['debian+buster', 'debian+bullseye', 'ubuntu+bionic', 'ubuntu+focal'];
+
+const getVersion = (versions, id) => versions[id] || 'master';
+
+const getToolLocation = ({ isPublic = true }) => (isPublic ? 'mender.io' : 'customer.mender.io/content/hosted');
+
+const defaultLocationFormatter = ({ os, tool, versionInfo }) => {
+  const location = getToolLocation(tool);
+  const { id, osList = [], title } = tool;
+  if (osList.length) {
+    return osList.reduce((accu, supportedOs) => {
+      const title = Object.entries(osMap).find(entry => entry[1] === supportedOs)[0];
+      accu.push({
+        location: `https://downloads.${location}/${id}/${getVersion(versionInfo, id)}/${supportedOs}/${id}`,
+        title,
+        isUserOs: osMap[os] === supportedOs
+      });
+      return accu;
+    }, []);
+  }
+  return [{ location: `https://downloads.${location}/${id}/${getVersion(versionInfo, id)}/linux/${id}`, title }];
+};
+
+const multiArchLocationFormatter = ({ tool, versionInfo }) => {
+  const location = getToolLocation(tool);
+  const { archList, id, osList, packageId } = tool;
+  return osList.reduce((accu, os) => {
+    const osArchitectureLocations = archList.map(arch => ({
+      location: `https://downloads.${location}/repos/debian/pool/main/m/${id}/${encodeURIComponent(
+        `${packageId || id}_${getVersion(versionInfo, id)}-1+${os}_${arch}.deb`
+      )}`,
+      title: `${os} - ${arch}`
+    }));
+    accu.push(...osArchitectureLocations);
+    return accu;
+  }, []);
+};
+
+const nonOsLocationFormatter = ({ tool, versionInfo }) => {
+  const location = getToolLocation(tool);
+  const { id, title } = tool;
+  return [
+    {
+      location: `https://downloads.${location}/${id}/${getVersion(versionInfo, id)}/${id}-${getVersion(versionInfo, id)}.tar.xz`,
+      title
+    }
+  ];
+};
+
+const defaultCurlDownload = ({ location, tokens }) => {
+  let headers = `-H cookie: JWT=${getToken()}`;
+  if (tokens.length) {
+    headers = `-H Authorization: Bearer ${tokens[0]}`;
+  }
+  return `curl ${headers} -LO ${location}`;
+};
+
+const defaultGitlabJob = ({ location, tokens }) => {
+  const filename = location.substring(location.lastIndexOf('/') + 1);
+  return `
+download:mender-tools:
+  image: curlimages/curl
+  stage: download
+  variables:
+    ${tokens.length ? `MENDER_TOKEN: ${tokens}` : `MENDER_JWT: ${getToken()}`}
+  script:
+    - if [ -n "$MENDER_TOKEN" ]; then
+    - curl -H Authorization: Bearer $MENDER_TOKEN -LO ${location}
+    - else
+    - ${defaultCurlDownload(location)}
+    - fi
+  artifacts:
+    expire_in: 1w
+    paths:
+      - ${filename}
+`;
+};
+
+const tools = [
+  {
+    id: 'mender',
+    packageId: 'mender-client',
+    title: 'Mender Client package',
```

Comment:
Maybe "Debian package"?

```suggestion
    title: 'Mender Client Debian package',
```